src/mol-canvas3d/canvas3d.ts

```diff
@@ -124,7 +124,8 @@ namespace Canvas3DContext {
         pickScale: 0.25,
         /** extra pixels to around target to check in case target is empty */
         pickPadding: 1,
-        enableWboit: true,
+        enableWboit: false,
+        enableDpoit: true,
```

Comment:
Please throw and error in the constructor if both WBOIT and DPOIT are true.

Also, as far as I understand, the DPOIT is quite a bit slower, so we might keep the WBOIT as default